tests/Propel/Tests/Runtime/Util/ProfilerTest.php

```diff
@@ -209,7 +209,7 @@ public function providerForTestToPrecision()
             [123.4567890, number_format(123)],
             [12.34567890, number_format(12.3, 1)],
             [1.234567890, number_format(1.23, 2)],
-            [0, 0],
+            [0, '0'],
```

Comment:
hmm?

Reply:
Test cases are now properly displaying the expected return type.
The reason why one should always use assertSame() instead of assertEquals() in tests for scalar values.